### PR TITLE
Added wide printer.

### PR DESCRIPTION
**Pull request by hoshir**

### CODE REVIEW

File: util/printer/wide_printer.go [hunk 1]
```diff
@@ -0,0 +1,121 @@
+// Copyright © 2018 Infostellar, Inc.
```
Comment (anuraaga) [inline util/printer/wide_printer.go:1]: Did you copy any code?

Comment (hoshir) [reply util/printer/wide_printer.go:1]: No, I got the idea to use tabwriter from kubernetes code, but don't copy from it.

Comment (hoshir) [reply util/printer/wide_printer.go:1]: Comments of tabwriter parameters are from Go documents.

File: util/printer/wide_printer.go [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+	// Formatting control.
+	// See possible values in https://golang.org/pkg/text/tabwriter/
+	flags = 0
+	// Minimal cell width including any padding.
+	minWidth = 4
+	// Padding added to a cell before computing its width
+	padding = 2
+	// ASCII char used for padding.
+	padChar = ' '
+	//Width of tab characters (equivalent number of spaces)
```
Comment (anuraaga) [inline util/printer/wide_printer.go]: Space between `//` and `W`

**Review by anuraaga — APPROVED** [review]

**Pull request merged**